docker/docker.md

```diff
@@ -0,0 +1,87 @@
+# sim-ln Docker Setup
+
+This guide provides instructions on how to build, run, and manage the `sim-ln` Docker container using the provided Makefile.
+
+## Prerequisites
+
+- Docker installed on your machine.
+- A `config.json` file that contains your specific configuration.
+
+## Building the Docker Image
+
+To build the Docker image, execute:
+
+```bash
+make build-docker
+```
+
+This command will make the necessary preparations and build the Docker image.
+
+## Mounting the Volume
+
+To ensure that the Docker container can access the configuration and other necessary data, we use Docker volumes. Before you can run the container, you need to set up the volume with the necessary data.
+
+To do this:
+
+```bash
+make mount-volume CONFIG_PATH=/path/to/your/config.json
+```
+
+Replace `/path/to/your/config.json` with the actual path to your `config.json` file.
+
+*Note:* The path to your config must be an absolute path not relative e.g. "/Users/anonymous/bitcoin-dev-project/sim-ln/config.json"
+
+## Running the Docker Container
+
+To run the `sim-ln` Docker container:
+
+```bash
+make run
+```
+
+You can adjust the logging level by providing the `LOG_LEVEL` variable. The default value is `info`. Example:
+
+```bash
+make run LOG_LEVEL=debug
+```
+
+Other configurable variables include:
+
+- `HELP`: Set to `true` to print the help message.
+- `PRINT_BATCH_SIZE`: Set the batch size for printing the results.
+- `TOTAL_TIME`: Adjusts the total time.
```

Comment:
nits on the variable docs: 
- `PRINT_BATCH_SIZE`: determines the number of payment results that will be written to disk at a time. 
- `TOTAL_TIME`: the total runtime for the simulation expressed in seconds.